aws/resource_aws_iam_role.go

```diff
@@ -120,6 +124,34 @@ func resourceAwsIamRole() *schema.Resource {
 				Default:      3600,
 				ValidateFunc: validation.IntBetween(3600, 43200),
 			},
+
+			"managed_policy_arns": {
+				Type:     schema.TypeSet,
+				Optional: true,
+				Computed: true,
+				Elem:     &schema.Schema{Type: schema.TypeString},
+				Set:      schema.HashString,
+			},
+
+			"inline_policies": {
+				Type:             schema.TypeMap,
+				Optional:         true,
+				Computed:         true,
+				DiffSuppressFunc: resourceAwsIamRoleSuppressPolicyDiffs,
+				Elem:             &schema.Schema{Type: schema.TypeString},
+			},
+
+			"managed_policies_exclusive": {
+				Type:     schema.TypeBool,
+				Optional: true,
+				Computed: true,
+			},
+
+			"inline_policies_exclusive": {
```

Comment:
We generally discourage the usage of sidecar attributes like this - being that `inline_policies` is `Optional: true` (you aren't required to opt into exclusive management) and `Computed: true` (we can ignore the difference if not configured) should be enough control of the situation.

Reply:
Never mind. There are no more sidecar attributes!